tests/Main.idr

```diff
@@ -203,17 +216,17 @@ runTest opts testPath
                            else print FileNotFound
                          pure False
                      | Left err => do print err
-                                      pure False
-
-                 if (out == exp)
+                                      pure False 
+                 let result = normalize out == normalize exp
+                 if normalize out == normalize exp 
```

Comment:
Here I should of course use `result`in the if-condition. But I can't! It bombs with a really weird error.

Reply:
```
Main.idr:222:26--223:21:While processing right hand side of runTest at Main.idr:166:1--231:1:
While processing right hand side of runTest,runTest' at Main.idr:206:9--231:1:
Can't solve constraint between:
        ()
and
        ?a [locals in scope: testPath, opts, {_:1513}, {_:1522}, out, {_cn:1}, exp]
```

Reply:
I get the same if I ditch normalize and just put in `out == exp` in result.